tests/test_entities.py

```diff
@@ -478,36 +478,47 @@ class CreateMissingTestCase(TestCase):
     def setUpClass(cls):
         """Set a server configuration at ``cls.cfg``."""
         cls.cfg = config.ServerConfig('http://example.com')
+        # Fields optionally populated by AuthSourceLDAP.create_missing()
+        cls.AS_LDAP_FIELDS = (
+            'account_password',
+            'attr_firstname',
+            'attr_lastname',
+            'attr_login',
+            'attr_mail',
+        )
 
     def test_auth_source_ldap_v1(self):
-        """Test ``AuthSourceLDAP(onthefly_register=False)``."""
+        """Test ``AuthSourceLDAP(onthefly_register=False).create_missing()``"""
         entity = entities.AuthSourceLDAP(self.cfg, onthefly_register=False)
-        with mock.patch.object(EntityCreateMixin, 'create_raw'):
-            with mock.patch.object(EntityReadMixin, 'read_raw'):
-                entity.create_missing()
-        self.assertEqual(
-            _get_required_field_names(entity).union(('onthefly_register',)),
-            set(entity.get_values().keys()),
+        with mock.patch.object(EntityCreateMixin, 'create_missing'):
+            entity.create_missing()
+        self.assertTrue(
+            set((self.AS_LDAP_FIELDS)).isdisjoint(entity.get_values())
```

Comment:
I had completely failed to recognize that Python's `set` function has powerful methods like `issubset` and `isdisjoint` when writing many of the tests in `CreateMissingTestCase`. The tests herein can probably be significantly simplified by taking advantage of these methods.